src/test/java/roomescape/time/service/ReservationTimeServiceTest.java

```diff
@@ -80,34 +91,45 @@ void setUp() {
+        assertThatThrownBy(() -> reservationTimeService.deleteById(1L)).isInstanceOf(NoSuchElementException.class)
+                .hasMessage("예약 시간을 찾을 수 없습니다.");
     }
 
     @Test
     void 이미_해당_시간의_예약이_존재한다면_삭제할_수_없다() {
-        Reservation reservation = reservationDbFixture.예약_유저1_내일_10시_공포();
-
-        assertThatThrownBy(() -> reservationTimeService.deleteById(reservation.getReservationTime().getId()))
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessage("[ERROR] 해당 시간에 이미 예약이 존재하여 삭제할 수 없습니다.");
+        Member reserver = memberDbFixture.유저1_생성();
+        ReservationDateTime reservationDateTime = reservationDateTimeDbFixture.내일_열시();
+        Theme theme = themeDbFixture.공포();
+
+        Reservation reservation = reservationRepository.save(Reservation.reserve(reserver, reservationDateTime, theme));
+        assertThatThrownBy(
+                () -> reservationTimeService.deleteById(reservation.getReservationTime().getId())).isInstanceOf(
+                InvalidArgumentException.class).hasMessage("해당 시간에 이미 예약이 존재하여 삭제할 수 없습니다.");
     }
 
     @Test
     void 예약_가능한_시간을_조회한다() {
-        Reservation reservation1 = reservationDbFixture.예약_유저1_내일_10시_공포();
-        reservationDbFixture.예약_유저1_내일_11시_공포();
-        AvailableReservationTimeRequest request = new AvailableReservationTimeRequest(
-                reservation1.getDate(),
-                reservation1.getTimeId()
-        );
+        Member reserver = memberDbFixture.유저1_생성();
+        ReservationDateTime reservationDateTime1 = reservationDateTimeDbFixture._7일전_열시();
+        ReservationDateTime reservationDateTime2 = reservationDateTimeDbFixture.내일_열한시();
+        Theme theme = themeDbFixture.공포();
+
+        Reservation reservation1 = reservationRepository.save(
+                Reservation.reserve(reserver, reservationDateTime1, theme));
+        reservationRepository.save(Reservation.reserve(reserver, reservationDateTime2, theme));
+
+        AvailableReservationTimeRequest request = new AvailableReservationTimeRequest(reservation1.getDate(),
+                reservation1.getTimeId());
 
         List<AvailableReservationTimeResponse> responses = reservationTimeService.getAvailableReservationTimes(request);
 
-        assertThat(responses.get(0).id()).isNotNull();
-        assertThat(responses.get(0).startAt()).isEqualTo(reservation1.getReservationTime().getStartAt());
-        assertThat(responses.get(0).isReserved()).isEqualTo(true);
-        assertThat(responses.get(1).isReserved()).isEqualTo(false);
+        SoftAssertions softly = new SoftAssertions();
+
+        softly.assertThat(responses.get(0).id()).isNotNull();
+        softly.assertThat(responses.get(0).startAt()).isEqualTo(reservation1.getReservationTime().getStartAt());
+        softly.assertThat(responses.get(0).isReserved()).isEqualTo(true);
+        softly.assertThat(responses.get(1).isReserved()).isEqualTo(false);
+
+        softly.assertAll();
```

Comment:
```suggestion
        SoftAssertions.assertSoftly(softly -> {
            softly.assertThat(responses.get(0).id()).isNotNull();
            softly.assertThat(responses.get(0).startAt()).isEqualTo(reservation1.getReservationTime().getStartAt());
            softly.assertThat(responses.get(0).isReserved()).isEqualTo(true);
            softly.assertThat(responses.get(1).isReserved()).isEqualTo(false);
        });
```

위와 같은 방식이 검증 요소의 응집도를 높일 수 있어서 좋다고 생각하는데, 폰트는 어떻게 생각하시는지 궁금해요~

Reply:
너무 깔끔하네요. 저도 동의합니다!